### PR TITLE
OO-de_de_update

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

File: i18n/de_DE.po [hunk 1]
```diff
@@ -2680,7 +2662,7 @@ msgstr ""
 
 #: app/scripts/arbeitsangebote/detail/duplicate/duplicate-arbeitsangebot.html:6
 msgid "Rythmus"
-msgstr ""
+msgstr "Rhythmus"
```
Comment (robstoll) [inline i18n/de_DE.po:2684]: fix in de_CH as well

File: i18n/de_DE.po [hunk 2]
```diff
@@ -1311,19 +1288,19 @@ msgstr ""
 
 #: app/scripts/kunden/detail/kundendetail.html:486
 msgid "Faelligkeitsdatum"
-msgstr ""
+msgstr "Fälligkeitsdatum"
 
 #: app/scripts/app.js:172
 msgid "FaelltAusAbwesend"
-msgstr ""
+msgstr "FälltAusAbwesend"
```
Comment (robstoll) [inline i18n/de_DE.po]: could be (also for de_CH)
```suggestion
msgstr "Fällt aus: Abwesend"
```

File: i18n/de_DE.po [hunk 3]
```diff
@@ -1311,19 +1288,19 @@ msgstr ""
...
 
 #: app/scripts/app.js:174
 msgid "FaelltAusGekuendigt"
-msgstr ""
+msgstr "FälltAusGekündigt"
```
Comment (robstoll) [inline i18n/de_DE.po]: could be (also for de_CH)
```suggestion
msgstr "Fällt aus: Gekündigt"
```

File: i18n/de_DE.po [hunk 4]
```diff
@@ -1311,19 +1288,19 @@ msgstr ""
...
 
 #: app/scripts/app.js:173
 msgid "FaelltAusSaldoZuTief"
-msgstr ""
+msgstr "FälltAusSaldoZuTief"
```
Comment (robstoll) [inline i18n/de_DE.po]: could be (also for de_CH)
```suggestion
msgstr "Fällt aus: Saldo zu tief"
```

File: i18n/de_DE.po (hunk 1 above)
Comment (mcmpp) [reply i18n/de_DE.po:2684]: done

File: i18n/de_DE.po (hunk 2 above)
Comment (mcmpp) [reply i18n/de_DE.po]: done

File: i18n/de_DE.po (hunk 3 above)
Comment (mcmpp) [reply i18n/de_DE.po]: done

File: i18n/de_DE.po (hunk 4 above)
Comment (mcmpp) [reply i18n/de_DE.po]: done

File: i18n/de_DE.po [hunk 5]
```diff
@@ -103,55 +98,60 @@ msgstr "+49..."
...
+
 #: app/scripts/arbeitsangebote/detail/arbeitsangebotedetail.html:75
 msgid "<button ng-if=\"!arbeitsangebotForm.$invalid\" type=\"button\" role=\"button\" ng-click=\"aaMultiplizieren()\" class=\"btn btn-default\"><span class=\"glyphicon glyphicon-plus\"></span> <span translate=\"\">Multiplizieren</span></button>"
 msgstr ""
 
-#: app/scripts/abos/detail/abosdetail.html:189
-msgid "<input id=\"zusatzaboPrice\" class=\"col-md-9\" type=\"number\" step=\"0.01\" ng-model=\"zusatzAbo.price\">\n"
-"                    <div ng-if=\"zusatzAbo.price != zusatzAbo.zusatzaboTypPrice\" class=\"col-md-1\"><button type=\"button\" class=\"btn btn-xs\" ng-click=\"defaultZusatzaboPrice(zusatzAbo.id)\" title=\"{{'Den Standardpreis wiederherstellen'|translate}}\"><span class=\"glyphicon glyphicon-erase\"></span> </button>\n"
-"                    </div>"
+#: app/scripts/reports/detail/reportsdetail.html:14
```
Comment (robstoll) [inline i18n/de_DE.po]: This looks suspicious as there isn't a template change in addition. On purpose?

Also, ability to have a different icon per language on purpose?

Comment (mcmpp) [reply i18n/de_DE.po]: done

**Review by robstoll — APPROVED** [review]

**Review by robstoll — APPROVED** [review]

**Pull request merged**